src/app/services/locales/en-us/index.yaml

```diff
@@ -9,5 +9,8 @@ services:
         service-config-view: 'YAML'
     items:
       title: Services
+      tray_title: "{name}"
   detail:
     no_matching_external_service: 'No matching ExternalService was found for service {name}'
+  overview: 'Overview'
+  config: 'YAML'
```

Comment:
Could these things be nested somehow, seems strange to me that we put the tray header/title in `services.routes.items.tray_title` but the sub headers/titles in `services.overview/config`.

Honestly looking at the commonalities here, we should probably reuse the terms from `services.routes.item`, theres possibly something to be said in certain cases for sometimes not reusing the route components for both the item and summary view, but for the actual terms we use in both of those, I'm pretty sure in the vast majority of cases we probably want to use the same text.

Reply:
I switched the tray’s title out for the item title, but I’m not comfortable using the `navigation` key entries seeing as they’re keyed with route names.

Reply:
Hmm yeah thats a good point! Could we maybe move these last ones to `services.routes.item.overview/config` then? That way they are all in the same place. I think it was said a while back that general it didn't make sense for some i18n to come form one places and some from another for no apparent reason.

Reply:
At this point I’m not sure why we introduced the `routes.item[s]` structures. What’s the point of putting all translations there? Might as well drop them put things on the top-level with minimal nesting. I can see some value in there being those `routes.item[s]` structures, but that vanishes if we just put everything on them that doesn’t relate to routes anymore.

Reply:
You're maybe right, but not sure, maybe here it doesn't seem that necessary but in other places it might. I think its just easier to be like "I'm in the item route, I need the stuff on the item property". I think also there are maybe cases where you'd have like `routes.items.introduction` and `routes.item.introduction` if we were to have like a short one sentence introductory sentence at the top of each view.

I think for the sake of these two bits of text though, shall we just follow whats there for now? We can chat about the i18n at a later date, I think at some point soon we should do a "i18n" effort similar to what we did with out "e2e testing" effort.

I suppose I'm not too precious over this one, so up to you.

Reply:
I moved those now.